docs/reference/migration/migrate_7_8.asciidoc

```diff
@@ -92,6 +92,35 @@ To avoid deprecation warnings, discontinue use of the `node.local_storage`
 setting.
 ====
 
+[[deprecate-basic-license-feature-enabled]]
+
+.In {es} 7.8.0, the following settings no longer have any effect, and
+have been deprecated:
+
+* `xpack.enrich.enabled`
+* `xpack.flattened.enabled`
+* `xpack.ilm.enabled`
+* `xpack.monitoring.enabled`
+* `xpack.rollup.enabled`
+* `xpack.slm.enabled`
+* `xpack.sql.enabled`
+* `xpack.transform.enabled`
+* `xpack.vectors.enabled`
+
+[%collapsible]
+====
+*Details* +
+Previously, these settings could be set to `false` in order to disable the
+feature's APIs in a cluster. As of 7.8.0, these basic license features are
+always enabled for the {default-dist}.
```

Comment:
The short description has to be a one liner. The following change should render correctly.

```suggestion
.Several {xpack} settings no longer have any effect and are deprecated. 

[%collapsible]
====
*Details* +
Basic {xpack} license features are always enabled for the {default-dist} 
and the following settings no longer have any effect: 

* `xpack.enrich.enabled`
* `xpack.flattened.enabled`
* `xpack.ilm.enabled`
* `xpack.monitoring.enabled`
* `xpack.rollup.enabled`
* `xpack.slm.enabled`
* `xpack.sql.enabled`
* `xpack.transform.enabled`
* `xpack.vectors.enabled`

Previously, they could be set to `false` to disable the feature's APIs in a cluster.
```